libraries/lib-wave-track/Sequence.cpp

```diff
@@ -778,41 +773,6 @@ void Sequence::AppendBlock( SampleBlockFactory *pFactory, sampleFormat format,
    // function gets called in an inner loop.
 }
 
-sampleCount Sequence::GetBlockStart(sampleCount position) const
-{
-   int b = FindBlock(position);
-   return mBlock[b].start;
-}
-
-size_t Sequence::GetBestBlockSize(sampleCount start) const
-{
-   // This method returns a nice number of samples you should try to grab in
```

Comment:
Finally I reach the commit that deletes the two functions above, and much else.

As I already explained, I don't like these deletions from Sequence.  I think some valuable old logic for alignment of cache fetches is lost.  I think these two functions at least should be retained, and they should be private, but the cache of the sequence should be a friend or a nested class.